.github/workflows/pytest_asdf.yml

```
@@ -1,19 +1,24 @@
 name: pytest asdf
-on: [push, pull_request]
+on:
+  push:
+    paths:
+      - 'weldx/**'
+      - '.github/workflows/pytest_asdf.yml'
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+    paths:
+      - 'weldx/**'
+      - '.github/workflows/pytest_asdf.yml'
```

```suggestion
on:
  push:
    paths:
      - 'weldx/**'
      - 'tutorials/**'
      - '.github/workflows/pytest_asdf.yml'
  pull_request:
    types: [opened, synchronize, reopened, ready_for_review]
    paths:
      - 'weldx/**'
      - 'tutorials/**'
      - '.github/workflows/pytest_asdf.yml'
```

Since our tutorials are also tested on Ubuntu (3.8?) we should include that directory. Maybe it is possible to include it only on that specific run

---

added that for pytest